autoscheduler/scraper/management/commands/scrape_courses.py

```diff
@@ -1,81 +1,108 @@
 import asyncio
-import time
+#import time
+import datetime
 from typing import List
 from django.core.management import base
 from scraper.banner_requests import BannerRequests
 from scraper.models import Course, Instructor, Section, Meeting
 from scraper.models.course import generate_course_id
-from scraper.models.section import generate_meeting_id, generate_meeting_time
+from scraper.models.section import generate_meeting_id
 
-def parse_section(courseData,instructor_object):
+def generate_meeting_time(string_time: str):
+    """ Generates a meeting time from a string in format hhmm.
+            ex) 1245 = 12:45am. 1830 = 6:30 pm."""
+    if string_time == "":
+        return None
+    hour = int(string_time[0:2])
+    minute = int(string_time[2:4])
+    meeting_time = datetime.time(hour, minute)
+
+    return meeting_time
+
+def generate_meeting_days(meetings_data):
```

Comment:
Similar to the above comment, I think a better name for this would be `parse_meeting_days`, since it goes through all of the `day` fields and creates an object out of it (which is what the other `parse_` functions do, expect they make actual Models)

Reply:
See above comment.